Classes/common/touchdb/TDRemoteRequest.m

```diff
@@ -174,80 +183,22 @@ - (BOOL)retry
     // request might have succeeded on the remote server, and by retrying we'd be issuing it again.
     // PUT and POST requests aren't generally idempotent, but the ones sent by the replicator are.
 
-    if (_retryCount >= kMaxRetries) return NO;
-    NSTimeInterval delay = RetryDelay(_retryCount);
-    ++_retryCount;
-    CDTLogVerbose(CDTTD_REMOTE_REQUEST_CONTEXT, @"%@: Will retry in %g sec", self, delay);
-    [self startAfterDelay:delay];
-    return YES;
+//    if (_retryCount >= kMaxRetries) return NO;
```

Comment:
There's a load of deleted / commented out code in here around retries and authentication. I think we need to figure out if it's a bad thing to lose this functionality and what functionality can be done by the filterchain and what can be done by NSURLSession